account_invoice_triple_discount/models/account_move.py

```diff
@@ -16,6 +16,8 @@ def _recompute_tax_lines(self, **kwargs):
         restored after the original process is done
         """
         old_values_by_line_id = {}
+        digits = self.line_ids._fields["price_unit"]._digits
```

Comment:
Please add a comment indicating why you are doing this for later references without having to do git blame and similar "arcane techniques".